server/src/main/java/org/elasticsearch/common/xcontent/support/XContentMapValues.java

```diff
@@ -332,6 +332,13 @@ public static String nodeStringValue(Object node, String defaultValue) {
         return node.toString();
     }
 
+    public static String nodeStringValue(Object node) {
```

Comment:
Its probably worth adding javadoc about how this is different from `Objects.toString`.

Reply:
++